BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.m

```diff
@@ -60,10 +58,9 @@ - (BOXAPIOperation *)createOperation
     if (self.requestAllFileFields) {
         fieldString = [self fullFileFieldsParameterString];
     }
-    
-    NSString *repsFieldString = [self.repsHelper getRepresentationsFieldString];
-    if(repsFieldString.length > 0) {
-        fieldString = [fieldString stringByAppendingFormat:@",%@", repsFieldString];
+
+    if(self.representationHints.length > 0) {
+        fieldString = [fieldString stringByAppendingFormat:@",%@,%@", BOXAPIObjectKeyRepresentations, BOXAPIObjectKeyAuthenticatedDownloadURL];
```

Comment:
Why do we always want to request authenticated_downloadURL?

Reply:
We used to do that in `getRepresentationsFieldString` based on the contents of the representationOptions. We no longer have that signal. The idiomatic way to handle this would be to add authenticatedDownloadURL to allFields, and just get it all the time, but this was a more conservative change.